public/locales/hu/common.json

```diff
@@ -0,0 +1,22 @@
+{
+    "logo": {
+        "alt": "Az Instant Visio logója, amely egy kék színű mobiltelefont ábrázol, amelyben megjelenik egy arc egy narancssárga kör formájában, amely épp videókonferenciában társalog a telefon birtokosával, akinek arca egy narancssárga körvonalakkal rendelkező kör formájában jelenik meg."
+    },
+    "homeBaseline": "Érje el ismerőseit videókonferencián, egy kattintással, <1>ingyenesen.</1>",
+    "headerBaseline": "Érje el ismerőseit videókonferencián, egy kattintással, ingyenesen.",
+    "footer": {
+        "contact-us": "Kapcsolat",
+        "legal-mentions": "Jogi közlemények",
+        "personal-data": "Személyes adatok",
+        "blog": "Blog",
+        "credits": "Közreműködők",
+        "contact": "Kapcsolatfelvétel"
+    },
+    "url": {
+        "legal-mentions": "mentions-legales",
+        "personal-data": "donnees-personnelles",
+        "blog": "blog",
+        "blog-article": "blog/:article",
```

Comment:
As changes were since made to the French version of this file (available here: https://github.com/Instant-Visio/InstantVisio-WebApp/blob/81ab76abe8653c9f972035ff87b4bccf7f3d1e6b/public/locales/de/common.json), could you, in the url object:

remove the "blog-article" key
add "video-call": "visio"